### PR TITLE
fix raw codec number to be 0x55

### PR DESCRIPTION
**Pull request by whyrusleeping**
cc @Kubuxu @diasdavid to double check the rest

### CODE REVIEW

**Comment by Kubuxu** [comment]
There is no dag-json format in multicodec table.

File: cid.go [hunk 1]
```diff
@@ -14,9 +14,10 @@ import (
 const UnsupportedVersionString = "<unsupported cid version>"
 
 const (
+	Raw = 0x55
```
Comment (diasdavid) [inline cid.go]: 👍

File: cid.go [hunk 2]
```diff
@@ -14,9 +14,10 @@ import (
...
+
 	Protobuf = 0x70
 	CBOR     = 0x71
-	Raw      = 0x72
 	JSON     = 0x73
```
Comment (whyrusleeping) [inline cid.go]: Ehh... probably not.

**Comment by kevina** [comment]
Um, this is a breaking change.  After this change when I tried to use raw leaves I now get unrecognized object type.

Is there any way to fix an repo with the incorrect numbers?

**Comment by whyrusleeping** [comment]
It's a breaking change only between two unreleased versions. You can do a
file renaming in the blocks directory to fix it.

**Pull request merged**